VaticanBAV/et/BAVet119.xml

```diff
@@ -0,0 +1,552 @@
+                                  <choice><sic>ይሠርቀ</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ይሠርቅ፡</corr></choice> ፳፫<supplied reason="omitted">፡</supplied> ያበርህ፡ ዕለት፡ 
+                                  <surplus>ዕለት፡</surplus> ቤተ፡ ፀሩ፡ አልኬድ፡ ፪፭፮፯፲ ወ ፩፲ ወ፪<supplied reason="omitted">፡</supplied> ቤተ፡ ክፍሉ<supplied reason="omitted">፡</supplied>
+                                  ፩፫፬፰፱፲ወ፬፲ወ፭ወ፲ወ፮<supplied reason="omitted">፡</supplied> ኮከቡ<supplied reason="omitted">፡</supplied> እልመሸተሪ፡ ዕለ<supplied reason="omitted">ቱ፡
+                                  ሐ</supplied> <supplied reason="omitted">ሙስ፡</supplied>
+                                   </incipit>      
+                          </msItem>
+                          <msItem xml:id="ms_i2.2">
+                              <locus target="#38r"/>
+                              <title type="complete" ref="LIT7107Divination#table2">Table containing the names of the sixteen constellations of the previous table and some directives on their use.</title>
+                              <incipit xml:lang="gez">
+                                  ለፋል፡ ዘይትገደፍ፡ ለ<add place="above">ነ</add>ቢር<supplied reason="omitted">፡</supplied> ስም፡ ወእም፡ ወምድር<supplied reason="omitted">፡</supplied> 
+                                  ለኮከብኒ፡ ስም፡ ወእም<supplied reason="omitted">፡</supplied> ለገይስ፡ ስም፡ ወእም<supplied reason="omitted">፡</supplied> ወርኅ፡ ዕለት፡ በ፲ወ፮ ግድፍ፡
+                              </incipit>      
+                          </msItem>
+                          <msItem xml:id="ms_i2.3">
+                              <locus from="38v" to="42r"/>
+                              <title type="complete" ref="LIT7107Divination#chapters">Sixteen chapters</title>
+                              <incipit xml:lang="gez">
+                                  ፩ ቀዳማዊ፡ ዐዐዐዐዐዐዐ አላዘያን፡ ዘወጽአ፡ ለሠናይ፡ ዝንቱ፡ ቤት፡ ቤተ፡ ነፍስ፡ ወሕይወት፡ ወዝንቱ፡ ትእምርት፤ ኅሩይ፡ <sic>እሳቲዊ፡</sic> ወከዋክብት፡ አልመሻታሪ፡ 
+                                  <sic>ወእመዋዕል፡</sic> ሐሙስ፡ ወእግዚአብሔር፡ <sic>የአምር፡</sic> በአሰንዮ፡ ንብረት፡ ወኵሉ፡ ፍጥረት፡ ርቱዕ፡ ወብዙኅ፡ <sic>ፍሥሐ፡</sic> ወሐሤት፡ ወፍጻሜ፡ 
+                                  መፍቅድ፡ <sic>ወጸሕቅ፡</sic> <choice><sic>ወአግርር፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወአግርሮ፡</corr></choice>
+                                  ፀላእት፤ <choice><sic>ወነጸሮ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ወነጽሮ፡</corr></choice> ሕልም፡ ወአስተጋብዖ፡ ሠናያት፡ እምለፌ፡ ወእምለፌ፡
+                                  ነጊደ፡ ኢየሩሳሌም፡ ወፍጻሜ፡ ኵሉ፡ <sic>ተምኔታተ<supplied reason="omitted">፡</supplied></sic>
+                              </incipit>                     
+                              </msItem>
+                          <msItem xml:id="ms_i2.4">
+                              <locus from="42r" to="49r"/>
+                              <title type="complete" ref="LIT7107Divination#kawakebt">Chapter on divination through the planets and the lunar houses, entitled <foreign xml:lang="gez">ሐሳበ፡ ከዋክብት፡  ወመናዝል፡</foreign>.</title>
```

Comment:
<title>Planets and lunar houses</title>
<note>Chapter on divination through the planets and the lunar houses, entitled <foreign xml:lang="gez">ሐሳበ፡ ከዋክብት፡  ወመናዝል፡</foreign>.</note>